app/src/main/java/org/fossasia/openevent/general/auth/EditProfileViewModel.kt

```diff
@@ -30,6 +30,9 @@ class EditProfileViewModel(
     private var updatedImageTemp = MutableLiveData<File>()
     var avatarUpdated = false
     var encodedImage: String? = null
+    lateinit var userAvatar: String
+
+    fun userAvatarInitialized() = this::userAvatar.isInitialized
```

Comment:
Why?

Reply:
to retain userAvatar when the fragment is destroyed

Reply:
I think the null check can work here?